frontend/pages/project-developer/new.tsx

```diff
@@ -0,0 +1,575 @@
+import { ChangeEvent, useState, useCallback } from 'react';
```

Comment:
Should this file be under `project-developers/new` instead, following the `investors/new` pattern instead?  

The problem I see with the current slug is that (although unlikely) if a project developer's name is `New` and they end up with the `new` slug, either their profile will be inaccessible or this form will.